co-log-polysemy/co-log-polysemy.cabal

```
@@ -78,7 +78,7 @@ library
                            Colog.Polysemy.Effect
 
   build-depends:       co-log-core ^>= 0.2.0.0
-                     , polysemy ^>= 0.2.1.0
+                     , polysemy >= 0.2.1.0 && < 0.4
```

I think it's completely okay to use the latest version of `polysemy` for now
```suggestion
                     , polysemy ^>= 0.3.0.0
```